docs/content/concepts/rollout-management.md

```diff
@@ -52,6 +52,10 @@ The cascading execution of the deployment groups is based on two thresholds that
 
 One of the main paradigms of Eclipse hawkBit is, that a Distribution Set represents the currently installed software of a device. Hence, a device can have only one Distribution Set assigned/installed at a time. With _Multi-Assignments_ enabled, this paradigm shifts. Multi-Assignments allows to assign multiple Distribution Sets to a device simultaneously, without cancelling each other. As a consequence, an operator can trigger multiple campaigns addressing the same devices in parallel. 
 
+### Action weight
+
+To differentiate between important and less important updates a property called _weight_ is used. When multi-assignments is enabled every action has a weight value between (and including) 0 and 1000. The higher the weight the more important is the assignment represented by the action. Also when defining a _rollout_ or an _auto-assignment_ and multi-assignments is enabled a weight value has to be provided. This value is passed to the actions created during the execution of these _rollouts_ and _auto-assignments_. If no weight was provided the highest value of 1000 is used instead. 
+
```

Comment:
@stefbehl - This would be an additional change in terms of Ui changes to let the user have the additional choice. Do we want to add this as an enhancement in future?

Reply:
Adding for reference : https://products.bosch-si.com/jira/browse/MECS-9620